CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ Table of Contents
 
 ### Improvements
 
+- Added 'target', 'rel' and 'xmlns' as attributes to invisible filters and 'scale' as attribute to style filter.
```

Comment:
Should we do two lines for that?

Reply:
I think it's a very small change, that will effect only very few users. I wasn't even sure if an entry was warranted.